Functions/Export-GomRepo.ps1

```diff
@@ -36,7 +35,32 @@ function Export-GomRepo {
         if($teams.Count -gt 0){
             $repo | Add-Member -MemberType NoteProperty -Name Teams -Value $teams
         }
-
+        
+        Write-Host "Looking for CODEOWNERS file in $repoName at .github/CODEOWNERS"
```

Comment:
`Write-Host` is fine for local debugging, prefer `Write-Verbose` in the context of the codestyle for this repo. 

See e303f2e97e034822bda59ab4c0aa107af1c61e9e for recent context. Echo to stdout before an _actual change_ is attempted. Verbose stream should be the dump for all "got here" style statements you'd want to browse in the GH action log to figure out why shit happened like it did.

Ditto lines 44,50 in this file & line 43 in `Deploy-GomRepo`

----

Note PowerShell has more [default streams](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_output_streams) than Unix (it's also extensible on top of the defaults, but that's another story). TL;DR: verbose stream the "_got here_" messages and stdout the _actual changes_.